### ISSUE TITLE
Game Crashes Consistently on Launch

### ISSUE THREAD

**Issue by Rbmukthegreat** [spelunky-fyi/overlunky#141]
Hello, I have tried turning off my antivirus, and that continued to make the game crash. I have tried opening overlunky and the game at different times, and that still makes the game crash. I have tried to download and run overlunky on another pc with no antivirus on it, and overlunky makes spelunky crash on that pc. I have also tried running both spelunky and overlunky in admin mode, and that didn't work either. I have also tried being in game when launching overlunky. 

I have also tried using the latest wine download, and that also failed.

I used visual studio to debug what is happening, and I found this:
Unhandled exception at 0x00007FF808B5A4CF (injected.dll) in spel2_crashdump_2021_08_22T16_55_33.dmp: 0xC0000005: Access violation reading location 0x0000000000000000.

I assume this has something to do with getting the module base wrong or finding the wrong offsets.

If there is something that you recommend doing to fix this issue, please tell me.

**Comment by Dregu** [comment]
I have been seeing an increased amount of crash on start reports lately... Even I get these crashes, but more like once a week.

I don't really have any ideas, just curious if

- you are on Windows 10
- you are running unmodified Spelunky 2 1.21.0c
- you are running on Steam or the Goldberg emulator
- you are running WHIP or stable version
- OL worked for you in previous versions
- 0.5.3 works
- Playlunky works
- you get any useful output from Overlunky.exe in console

You can run OL and the game with some debug output in cmd:
```
...\Spelunky 2> set OL_DEBUG=1
...\Spelunky 2> Overlunky\Overlunky.exe --launch_game .
```

**Comment by Rbmukthegreat** [comment]
I am on Windows 10, I am running unmodified Spelunky 2 1.21.0c, I am using the Goldberg emulator, I tried both the stable and the whip versions, I have never used OL in previous versions as I just got the game recently in July, and 0.5.3 does work for me. I have not tried using Playlunky, and it says error: allocation failed 0x6 when I run the command you posted there. 

Anyways, 0.5.3 seems to work, so I will be using that for now.

Thank you!
